db/migrate/20220110090014_create_tickets.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTickets < ActiveRecord::Migration[7.0]
+  def change
+    create_table :tickets do |t|
+      t.string :seat
+      t.string :type
+      t.float :price
```

Comment:
Good article why you shouldn't store prices in float
https://medium.com/@cancerian0684/which-data-type-would-you-choose-for-storing-currency-values-like-trading-price-dd7489e7a439